fortnite_api/shop.py

```diff
@@ -140,30 +192,49 @@ def __init__(self, data):
 
 class BrShopSectionNew:
 
-    def __init__(self, data):
-        self.id = data.get('id')
-        self.name = data.get('name')
-        self.index = data.get('index')
-        self.landing_priority = data.get('landingPriority')
-        self.sort_offers_by_ownership = data.get('sortOffersByOwnership')
-        self.show_ineligible_offers = data.get('showIneligibleOffers')
-        self.show_ineligible_offers_if_giftable = data.get('showIneligibleOffersIfGiftable')
-        self.show_timer = data.get('showTimer')
-        self.enable_toast_notification = data.get('enableToastNotification')
-        self.hidden = data.get('hidden')
+    __slots__: Tuple[str, ...] = (
+        'id',
+        'name',
+        'index',
+        'landing_priority',
+        'sort_offers_by_ownership',
+        'show_ineligible_offers',
+        'show_ineligible_offers_if_giftable',
+        'show_timer',
+        'enable_toast_notification',
+        'hidden',
+    )
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.id: str = data['id']
+        self.name: str = data['name']
+        self.index: int = data['index']
+        self.landing_priority: int = data['landingPriority']
+        self.sort_offers_by_ownership: bool = data['sortOffersByOwnership']
+        self.show_ineligible_offers: bool = data['showIneligibleOffers']
+        self.show_ineligible_offers_if_giftable: bool = data['showIneligibleOffersIfGiftable']
+        self.show_timer: bool = data['showTimer']
+        self.enable_toast_notification: bool = data['enableToastNotification']
+        self.hidden: bool = data['hidden']
```

Comment:
Hidden seems to not be always present. If it's not present, we can set it to True